server/routes/checkInformationRoutes.test.ts

```diff
@@ -555,6 +555,57 @@ describe('Check information routes tests', () => {
         expect($('.new-sentence-card:contains("Rape of a minor")').text()).toContain('SDS+')
       })
   })
+  it('GET /calculation/:nomsId/check-information should show exclusions with feature toggle on for single sentence', () => {
+    config.featureToggles.sdsExclusionIndicatorsEnabled = true
+
+    calculateReleaseDatesService.getUnsupportedSentenceOrCalculationMessages.mockResolvedValue(stubbedEmptyMessages)
+    userInputService.isCalculationReasonSet.mockReturnValue(true)
+    const singleSentencesAndOffencesWithExclusions = [
+      {
+        terms: [
+          {
+            years: 3,
+          },
+        ],
+        sentenceTypeDescription: 'SDS Standard Sentence',
+        caseSequence: 1,
+        lineSequence: 2,
+        caseReference: 'CASE001',
+        courtDescription: 'Court 1',
+        sentenceSequence: 1,
+        offence: {
+          offenceStartDate: '2021-01-04',
+          offenceEndDate: '2021-01-05',
+          offenceDescription: 'VIOOFFENCE',
+        },
+        sentenceAndOffenceAnalysis: 'NEW',
+        isSDSPlus: true,
+        hasAnSDSEarlyReleaseExclusion: 'VIOLENT',
+      } as AnalysedSentenceAndOffence,
+    ]
+    const model = new SentenceAndOffenceViewModel(
+      stubbedPrisonerData,
+      stubbedUserInput,
+      singleSentencesAndOffencesWithExclusions,
+      stubbedAdjustments,
+      false,
+      stubbedReturnToCustodyDate,
+      null,
+    )
+    checkInformationService.checkInformation.mockResolvedValue(model)
+    return request(app)
+      .get('/calculation/A1234AA/check-information')
+      .expect(200)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        const $ = cheerio.load(res.text)
+        expect($('.new-sentence-card:contains("VIOOFFENCE")').text()).toContain('Violent')
+        const noExclusionCard = $('.new-sentence-card:contains("No exclusion offence")')
+        expect(noExclusionCard.text()).not.toContain('Sexual')
+        expect(noExclusionCard.text()).not.toContain('Violent')
```

Comment:
I don't think you need these lines 603-5

Reply:
Amended as requested.